lib/unexpectedMitm.js

```diff
@@ -876,24 +893,18 @@ module.exports = {
                 }
                 var nextRequestDescriptionIndex = 0;
 
+                var exchangeParts = [];
```

Comment:
Can we find a better variable name that makes it obvious that it'll contain a mix of `{ spec, httpExchange }` objects and errors? Maybe something with `timeline`?

Reply:
Right, yes - at minimum it's not actually exchangeParts, it's more like conversationParts. But that works for me too - think part of the difficulty is this is a kind of checkpoint rather than where we might want it to ultimately end up.